src/pages/Individual/IndividualFilter/IndividualFilterBar/IndividualFilterBar.js

```diff
@@ -1,63 +1,151 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import styled from 'styled-components';
 
-function IndividualFilterItem({ range, grade, earningPercent }) {
+import { GRADE_BAR_RANGE, EARNING_BAR_RANGE } from '../filterRange';
+
+function IndividualFilterItem({
+  maxValue,
+  grade,
+  earningPercent,
+  handleFilter,
+}) {
+  const [startValue, setStartValue] = useState(0);
+  const [endValue, setEndValue] = useState(maxValue);
+  const [rangeText, setRangeText] = useState({ start: '', end: '' });
+
+  let startText = (grade && 'A+') || (earningPercent && 4);
+  let endText = (grade && 'D-') || (earningPercent && 24);
+
+  useEffect(() => {
+    setRangeText({ start: startText, end: endText });
+  }, []);
```

Comment:
```suggestion
  let startText = (grade && 'A+') || (earningPercent && 4);
  let endText = (grade && 'D-') || (earningPercent && 24);
  const [rangeText, setRangeText] = useState({ start: startText, end: endText });
```
초기값으로 바로 넣어줘도 될 것 같습니다

Reply:
startText랑 endText를 일반 let 변수로 선언해서 재할당하면서 사용하시는 이유가 있으신가요?

Reply:
@younuk23 아니요 특별한 이유는 없습니다. 다시 보니 불필요한 부분이어서 수정했습니다. 감사합니다!